### PR TITLE
Update Finnish translations

### PR DESCRIPTION
**Pull request by tuhatk**
## Checklist

- [X] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test if it works standalone, system.c especially
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: data/languages/finnish.txt [hunk 1]
```diff
@@ -618,274 +617,274 @@ Type:
...
 
 Are you sure that you want to disconnect your dummy?
-== 
+== Oletko varma, että haluat katkaista dummysi yhteyden?
 
 Welcome to DDNet
-== 
+== Tervetuloa DDNet:iin
 
 DDraceNetwork is a cooperative online game where the goal is for you and your group of tees to reach the finish line of the map. As a newcomer you should start on Novice servers, which host the easiest maps. Consider the ping to choose a server close to you.
```
Comment (tuhatk) [inline data/languages/finnish.txt:682]: > Consider the ping to choose a server close to you.

The pings are no longer shown by default. Should this message be changed?

File: data/languages/finnish.txt [hunk 2]
```diff
@@ -957,226 +956,226 @@ Statboard
...
 
 Use modern OpenGL
-== 
+== Käytä nykyaikaista OpenGL:ää
 
 Preinit VBO (iGPUs only)
-== 
+== Esialusta VBO (vain iGPUt)
 
 Multiple texture units (disable for MacOS)
```
Comment (tuhatk) [inline data/languages/finnish.txt:997]: Should this be macOS instead of MacOS?

Comment (def-) [reply data/languages/finnish.txt:997]: Yes, I'll fix that later.

File: data/languages/finnish.txt (hunk 1 above)
Comment (def-) [reply data/languages/finnish.txt:682]: Well, they are kind of indicated and shown after having joined the server, so I think it's ok to keep it.

**Pull request merged**